wafer/schedule/templates/wafer.schedule/edit_schedule.html

```diff
@@ -119,6 +119,9 @@ <h2>{% trans 'Bucket' %}</h2>
                     id="talk{{ talk.talk_id }}"
                     data-toggle="tooltip" data-placement="left"
                     title="{{ talk.title }}"
+                    {% if talk not in talks_unassigned %}
+                    hidden
```

Comment:
I think that should be indented, to match the rest of the template style

Reply:
yeah, fixed